### ISSUE TITLE
MD064 false-positive with <pre> tag containing empty line

### ISSUE THREAD

**Issue by senekor** [rvben/rumdl#578]
### rumdl version

0.1.72

### Installation method

Other

### Bug description

If a `<pre>` tag contains an empty line, rumdl will warn about later consecutive spaces, even though they're still part of the `<pre>` tag.

### Steps to reproduce

Create file:
```md
# MD046 false-positive reproduction

<pre>

hello  world
</pre>
```
Run `rumdl check --no-config` on it.

### Expected behavior

No "Multiple consecutive spaces" warning, since it's in a `<pre>` tag.

### Actual behavior

The warning: "Multiple consecutive spaces (2) found"

(And a warning about the inline HTML, but that's expected. Users may allow inline HTML in their configuration, in which case the false-positive is the only warning left.)

### Configuration

not relevant with `--no-config`

### Operating system

Linux

### Additional context

installation method: nix

**Comment by rvben** [comment]
Fixed in [v0.1.78](https://github.com/rvben/rumdl/releases/tag/v0.1.78). Thanks for the report, @senekor!
